spectree/spec.py

```diff
@@ -264,6 +264,16 @@ def _generate_spec(self):
             },
         }
 
+        if self.config.SERVERS:
+            spec["servers"] = [
+                {
+                    "url": server.url,
+                    "description": server.description,
+                    "variables": server.variables,
+                }
+                for server in self.config.SERVERS
+            ]
```

Comment:
Is it better to use `[server.dict() for server in self.config.SERVERS]`?

Reply:
It seems good - I used vars or something like this previously, but the test failed due to different key positions. But this is OK :-)